osquery/tables/yara/tests/yara_tests.cpp

```diff
@@ -80,4 +78,32 @@ TEST_F(YARATest, test_match_false) {
   // Should have 0 count
   EXPECT_TRUE(r["count"] == "0");
 }
+
+TEST_F(YARATest, should_skip_file) {
+  // pretty much any regular file should be scanned
+
+  EXPECT_FALSE(yaraShouldSkipFile("/any/file/here", S_IFREG));
+
+  // should skip osquery db LOCK
+
+  EXPECT_TRUE(yaraShouldSkipFile("/var/osquery/osquery.db/LOCK", S_IFREG));
+  EXPECT_TRUE(yaraShouldSkipFile("C:\\Program Files\\osquery\\osquery.db\\LOCK",
+                                 S_IFREG));
+
+  // any other LOCK can be scanned
+
+  EXPECT_FALSE(yaraShouldSkipFile("/Applications/some/LOCK", S_IFREG));
```

Comment:
Is there a property of the `LOCK` file that we are interested in / open-exclusive or similar? vs. the name of the file?

Reply:
Only based on path at the moment.